heron/tools/tracker/src/python/tracker.py

```diff
@@ -64,17 +66,18 @@ def sync_topologies(self) -> None:
     def on_topologies_watch(state_manager: StateManager, topologies: List[str]) -> None:
       """watch topologies"""
       topologies = set(topologies)
-      Log.info("State watch triggered for topologies.")
-      Log.debug("Topologies: %s", topologies)
-      cached_names = {t.name for t in self.get_stmgr_topologies(state_manager.name)}
-      Log.debug("Existing topologies: %s", cached_names)
-      for name in cached_names - topologies:
-        Log.info("Removing topology: %s in rootpath: %s",
-                 name, state_manager.rootpath)
-        self.remove_topology(name, state_manager.name)
-
-      for name in topologies - cached_names:
-        self.add_new_topology(state_manager, name)
+      Log.info(f"State watch triggered for topologies of {state_manager.name}.")
+      Log.debug(f"Received topologies: {state_manager.name}, {topologies}")
```

Comment:
it's generally recommended to leave the logging in the form `(format_str, *args)`, as it avoids unnecessary stringification and interpolation (i.e. when the message level is lower than the configured logging level), so would be good to change back to that format

Reply:
Updated

Reply:
```
bazel test --config=stylecheck heron/tools/tracker/...
```
```
C0209: Formatting a regular string which could be a f-string (consider-using-f-string)
```
Reverted

Reply:
is that the case for things like `Log.debug("Existing topologies: %s", cached_names")`? That is different from `"Existing topologies: %s" % cached_names` which is where I'd expect to see what linter warning

Reply:
Correct. I don't think this was a style check issue. When I did the conversion to f-string during the Python upgrade, I left some of these Log statements in the (format_str, &args) format because it is more efficient. Doing the f-string first and then passing into a log statement that might not render is wasted effort. The Log statement will only do the string interpolation if the log statement needs to be rendered. Because of this, I felt it was better to keep it in the original form.